tests/manager/manager-maxstalenessms_error-001.phpt

```diff
@@ -1,33 +0,0 @@
---TEST--
-MongoDB\Driver\Manager: maxStalenessMS
```

Comment:
This was renamed to `manager-ctor-read_preference-error-002.phpt`, where I added an additional error case for an out-of-range value.